docs/docs/howto/installation/run-docker.rst

```diff
@@ -49,34 +79,55 @@ To configure a property, for example, ``klaw.login.authentication.type=db``, set
   It is important to update the ``KLAW_CLUSTERAPI_ACCESS_BASE64_SECRET`` property with a new base64-encoded secret.
 
 2. Run Docker Compose
+``````````````````````````
 
-Run the following command to start the containers defined in the Docker Compose file:
+Once you have created the Docker Compose file, run the following command to start the containers defined in the file:
 
 ::
   
-  ``docker-compose -f .\dockerpcompose.yml up``
+  docker-compose -f .\dockerpcompose.yml up
```

Comment:
This format does not work on mac, you can keep it simple by just doing `docker-compose  -f dockerpcompose.yml` or even better tell the user to name the file `docker-compose.yml` so he has just to do `docker-compose up`

Reply:
The run is failing : 
```
docker-compose -f ./docker-compose.yml up
[+] Running 0/2
 ⠿ klaw-cluster-api Error                                                                             1.1s
 ⠿ klaw-core Error                                                                                    1.1s
Error response from daemon: failed to resolve reference "docker.io/library/klaw-cluster-api:latest": pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```
It's because you forgot the repo name in front of the image name : 'aivenoy'